### ISSUE TITLE
Landing page not working in Firefox

### ISSUE THREAD

**Issue by rgreinho** [request-yo-racks/landing-page#4]
The page stays blank and the console shows the following error messages:
```
TypeError: script is null[Learn More] www.requestyoracks.org:1:2132
TypeError: window.customElements is undefined[Learn More] www.requestyoracks.org:1:20574 
```

It works perfectly on Google Chrome and Safari.

System info:
```
Last commit:
  11fdd82 Fix polymer.json (#3)  (HEAD -> master, upstream/master, upstream/HEAD, origin/master)
Browser(s):
  Mozilla Firefox 61.0.1
  Google Chrome 67.0.3396.99 
  Safari: 11.1.1 (13605.2.8)
System Version: macOS 10.13.5 (17F77)
```

**Comment by rgreinho** [comment]
Digging a bit more, it looks like the polyfills are not loaded in Firefox. 

Chrome and Safari do not require any polyfill to use Custom Elements and Shadow DOM, that is why the landing page renders correctly on these browsers.

[These](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_shadow_DOM) [pages](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Using_custom_elements) mention these feature will be available in Firefox 63.

<img width="699" alt="image" src="https://user-images.githubusercontent.com/6969134/45191332-34caf980-b208-11e8-881c-b252400b175c.png">

<img width="699" alt="image" src="https://user-images.githubusercontent.com/6969134/45191345-490ef680-b208-11e8-9438-8ccce266b2f0.png">

**Comment by rgreinho** [comment]
Closed by #6 .
